infra/terraform/modules/lambda/main.tf

```diff
@@ -2,13 +2,22 @@ data "aws_iam_role" "tzlink_lambdas_access" {
   name = "tzlink_lambdas_access"
 }
 
-data "aws_instance" "tz_node" {
+data "aws_instances" "tz_nodes" {
   filter {
     name   = "tag:Name"
-    values = ["tzlink-mainnet-0"]
+    values = ["tzlink-mainnet"]
   }
+
+  instance_state_names = ["running"]
 }
 
+#data "aws_instance" "tz_node" {
```

Comment:
Still to keep ?

Reply:
Effectivement je m'en débarasse de suite

Reply:
Done